server/tserver/src/main/java/org/apache/accumulo/tserver/ThriftScanClientHandler.java

```diff
@@ -147,34 +147,40 @@ public void close() {}
     };
     return this.startScan(tinfo, credentials, extent, range, columns, batchSize, ssiList, ssio,
         authorizations, waitForWrites, isolated, readaheadThreshold, tSamplerConfig, batchTimeOut,
-        contextArg, executionHints, resolver, busyTimeout);
+        contextArg, executionHints, resolver, busyTimeout, userData);
   }
 
   public InitialScan startScan(TInfo tinfo, TCredentials credentials, KeyExtent extent,
       TRange range, List<TColumn> columns, int batchSize, List<IterInfo> ssiList,
       Map<String,Map<String,String>> ssio, List<ByteBuffer> authorizations, boolean waitForWrites,
       boolean isolated, long readaheadThreshold, TSamplerConfiguration tSamplerConfig,
       long batchTimeOut, String contextArg, Map<String,String> executionHints,
-      ScanSession.TabletResolver tabletResolver, long busyTimeout) throws NotServingTabletException,
-      ThriftSecurityException, org.apache.accumulo.core.tabletserver.thrift.TooManyFilesException,
+      ScanSession.TabletResolver tabletResolver, long busyTimeout, String userData)
+      throws NotServingTabletException, ThriftSecurityException,
+      org.apache.accumulo.core.tabletserver.thrift.TooManyFilesException,
       TSampleNotPresentException, ScanServerBusyException {
 
+    log.debug("({}) starting scan", userData);
+
     server.getScanMetrics().incrementStartScan(1.0D);
 
     TableId tableId = extent.tableId();
     NamespaceId namespaceId;
     try {
       namespaceId = server.getContext().getNamespaceId(tableId);
     } catch (TableNotFoundException e1) {
+      log.error("({}) not serving tablet {}", userData, extent.toThrift());
```

Comment:
Thinking this would be better logged at debug or trace.  Tables being concurrently deleted while a scan is attempted can be a normal course of business.